src/main/java/jalau/at18/katas/highestlowest/sarai/Main.java

```diff
@@ -0,0 +1,28 @@
+package jalau.at18.katas.highestlowest.sarai;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        int cant;
+        Scanner scanner_teclado = new Scanner(System.in);
```

Comment:
hi @saraialvarezrosas , please update your code to use english everywhere, this helps to keep consistency